Project Pathfinder/Assets/Scripts/CustomNetworkManager.cs

```diff
@@ -31,6 +31,10 @@ public override void OnClientConnect()
         base.OnClientConnect();
         NetworkClient.RegisterHandler<MazeMessage>(ReceiveMazeData);
         NetworkClient.RegisterHandler<AnimationMessage>(NetworkAnimationHandler);
+        
```

Comment:
This could be an issue. The way the network manager works is that first a server is created on the host machine, then the host joins said server as a client. This means OnClientConnect() will also run for the host (see documentation on Host Mode [here](https://mirror-networking.gitbook.io/docs/guides/communications/networkmanager-callbacks#host-mode). We can fix this potential issue by also doing a check on the client side to see if the client is also the host.